src/main/resources/db/migration/V6_9__move_pdf_columns.sql

```diff
@@ -0,0 +1,56 @@
+
+CREATE TABLE PDF
+(
+    id                           SERIAL PRIMARY KEY,
+    uuid                         VARCHAR(50)  NOT NULL UNIQUE,
+    created_at                   TIMESTAMP    NOT NULL,
+    updated_at                   TIMESTAMP    NOT NULL,
+    pdf                          bytea        NOT NULL
+);
+
+ALTER TABLE MOTE_REFERAT
+ADD COLUMN pdf_id INTEGER REFERENCES PDF (id) ON DELETE RESTRICT,
+ALTER COLUMN pdf DROP NOT NULL;
+
+ALTER TABLE MOTEDELTAKER_ARBEIDSTAKER_VARSEL
+ADD COLUMN pdf_id INTEGER REFERENCES PDF (id) ON DELETE RESTRICT,
+ALTER COLUMN pdf DROP NOT NULL;
+
+ALTER TABLE MOTEDELTAKER_ARBEIDSGIVER_VARSEL
+ADD COLUMN pdf_id INTEGER REFERENCES PDF (id) ON DELETE RESTRICT,
+ALTER COLUMN pdf DROP NOT NULL;
+
+ALTER TABLE MOTEDELTAKER_BEHANDLER_VARSEL
+ADD COLUMN pdf_id INTEGER REFERENCES PDF (id) ON DELETE RESTRICT,
+ALTER COLUMN pdf DROP NOT NULL;
+
+DO $$
+DECLARE v_id INTEGER;
+DECLARE v_row RECORD;
+BEGIN
+FOR v_row IN
+        SELECT uuid,pdf::bytea FROM MOTE_REFERAT
+    LOOP
+        INSERT INTO PDF VALUES ((overlay(overlay(md5(random()::text || clock_timestamp()::text) placing '4' from 13) placing '8' from 17)::uuid)::text,now(),now(),v_row.pdf::bytea) RETURNING id into v_id;
```

Comment:
Kan du forklare hva som skjer her? 😅

Reply:
Jeg fant noe kode for å generere uuid i db-script (finnes en egen postgres-funksjon for det, men det forutsetter en utvidelse som det ikke ser ut som vi har). Så det er det som gjør dette litt kryptisk, men resultatet skal bare være en unik uuid.